src/connector/src/base.rs

```diff
@@ -65,6 +66,12 @@ pub trait SplitReader: Sized {
     async fn next(&mut self) -> Result<Option<Vec<SourceMessage>>>;
 }
 
+pub const PULSAR_SPLIT_TYPE: &str = "pulsar";
```

Comment:
The code here has been moved to the corresponding connector folder, please rebase the main